CHANGELOG.md

```diff
@@ -56,6 +56,30 @@ Our versioning strategy is as follows:
 * `[angular]` Avoid sending two dictionary service calls when switching language and refreshing the page ([#1473](https://github.com/Sitecore/jss/pull/1473))
 * Fix installed sitecore-jss-* dependency version ([#1478](https://github.com/Sitecore/jss/pull/1478))
 
+## 21.1.5
+
+### 🐛 Bug Fixes
+
+* `[create-sitecore-jss]` Change ^ to ~ for versioning in templates and use exact versions for sitecore-jss monorepo dependencies.
+
+## 21.1.4
+
+### 🐛 Bug Fixes
+
+* `[create-sitecore-jss]` Change ^ to ~ for versioning in templates.
+
+## 21.1.3
+
+### 🐛 Bug Fixes
+
+*  Fix installed sitecore-jss-* dependency version. Change ^ to ~ ([#1481](https://github.com/Sitecore/jss/pull/1481))
+
+## 21.0.10
```

Comment:
This version should be below 21.1.0

Reply:
it should be 21.0.10 but should be below 21.1.x

Reply:
yeah, so needs to be moved below